### DISCUSSION TITLE
Community Meeting June 23

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#320] [Announcements]
Agenda items for our next community Zoom meeting, to be held on June 23, will be organized here.

- Defining spaces that are not unique up to homeomorphism. (#314)
  - Decision: "space schema" are okay, but can be specified later as a property that needs the specification is contributed.
  - Lynne's observation: spaces are themselves very specific properties. Steven: many spaces have characterizations as a combination of proper properties.
- "Hackathons", workshops, REUs
   - Invite folks in issues to co-author PRs.
   - Invite Ryan and DaRT into next community meeting discussion?
   - Connect with LMFDB community?
- AMS SE sectional meeting special session on math cyberinfrastructure [[Link](http://www.ams.org/meetings/sectional/2311_program.html)]
  - Twisted Chris's arm, maybe Hayden?
- Occasional "notify slack" errors in the build
  - will probably rip out that part of workflow

**Comment by ccaruvana** [comment]
Just as a heads up, I'll be generally unavailable on the 16th, but should be available the 30th, for what it's worth.

**Comment by StevenClontz** [comment]
> I'll be generally unavailable on the 16th

Let's punt to at least the 30th then - I'm slammed this week and won't be very prepared to run a meeting if we kept this Friday.

**Comment by StevenClontz** [comment]
Carolinas seminar is meeting on June 30. I'm available June 23 (unless they have dibs there too). My next available Friday is July 14.

**Reply by ccaruvana** [reply]
The 23rd works for me!

**Comment by prabau** [comment]
Ok for me too.

**Comment by lyengulalp** [comment]
Works for me.

**Comment by StevenClontz** [comment]
23rd it is

**Comment by prabau** [comment]
Item for the agenda: Occasional "notify slack" errors in the build

**Comment by ccaruvana** [comment]
Concerning the point about construction schema, according to the introduction and Figure 3 of https://www.sciencedirect.com/science/article/pii/S0166864196000752, one can build a special Luzin set which is Rothberger but **not** $\omega$-Rothberger, but that another construction generates Luzin sets that **are** $\omega$-Rothberger. Depending on whether https://github.com/pi-base/data/pull/329 gets merged, that would be an immediate example of a particular choice in the construction of a Luzin set that does distinguish between properties reflected in the pi-base (assuming the PR, no such space seems to be present).

**Reply by prabau** [reply]
> Concerning the point about construction schema, according to the introduction and Figure 3 of https://www.sciencedirect.com/science/article/pii/S0166864196000752, one can build a special Luzin set which is Rothberger but **not** ω-Rothberger, but that another construction generates Luzin sets that **are** ω-Rothberger. Depending on whether https://github.com/pi-base/data/pull/329 gets merged, that would be an immediate example of a particular choice in the construction of a Luzin set that does distinguish between properties reflected in the pi-base (assuming the PR, no such space seems to be present).

Interesting. Unfortunately, both the construction of the "special Luzin set" $L$ (lemma 2.6) and of the "generic Luzin set" $H$ (thm 2.13) depend on CH.

Still, if we could use that to illustrate assuming they were ZFC examples, one would have two spaces (or space schemas) showing the difference.

**Reply by ccaruvana** [reply]
> Interesting. Unfortunately, both the construction of the "special Luzin set" L (lemma 2.6) and of the "generic Luzin set" H (thm 2.13) depend on CH.

The existence of Luzin sets, in general, relies on CH to some degree, as their existence is independent of ZFC: https://en.wikipedia.org/wiki/Luzin_space

Maybe such independent constructions should be labeled as such?

**Reply by prabau** [reply]
That fits with Lynne's obervation that the line between spaces and properties is kind of fuzzy.  We cannot prove in ZFC that Luzin sets exist, but Luzin spaces can be made a property and then we can say some things about them. (not that I am advocating that, just an observation)

**Reply by ccaruvana** [reply]
This does seem to be especially relevant here since the original motivation was related to subsets of the real line (I think) which seems more property-oriented than space-related. At any rate, I would think it sensible to include these special spaces with some sort of alert tag that their existence may depend on certain set-theoretic axioms that are independent of ZFC.

In the case of Luzin sets, writing it in as a property instead of a class of spaces removes any assertions about existence, but there is some utility/interest in representing consistent examples, I think. Perhaps adding such tags that appear in the search feature to alert users that assumptions like CH are needed could be a useful compromise.

**Reply by prabau** [reply]
Yeah, maybe using these special tags at least for example spaces would be a good way to go.  We should add this as a topic for the next meeting.
